LayoutTests/accessibility/aria-combobox-no-owns.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../resources/js-test-pre.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+</head>
+<body id="body">
+
+<input type="text" role="combobox" id="combobox" aria-label="Combobox">
+<div role="list" id="list" aria-label="List1">
+  <div role="listitem" id="item1">item1</div>
+  <div role="listitem" id="item2">item2</div>
+</div>
+
+<p id="description"></p>
+<div id="console"></div>
+
+<script>
+    description("A combobox should still support aria-activedescendant even if it doesn't use aria-owns.");
```

Comment:
Rather than using `description`, our new layout tests would use:

```
var output = "A combobox should still support aria-activedescendant even if it doesn't use aria-owns.\n\n";
```

And rather than using `debug` and `shouldBe` throughout the test, we would do something like:

```
// debug("list has " + list.selectedChildrenCount + " selected children");
output += `list has ${list.selectedChildrenCount} selected children\n`;

// shouldBe("list.selectedChildrenCount", "0");
// This expect function is defined in accessibility-helpers.js
output += expect("list.selectedChildrenCount", "0");
```

And then at the end of the test, we print the output:

```
debug(output);
finishJSTest();
```

The reason for this change of style is because `description`, `debug`, and `shouldBe` write to the DOM every time, which we've found can significantly slow test execution. With this new style, we only write to the DOM once.

A good example to follow is:

https://github.com/WebKit/WebKit/blob/eb3708646ee56c088e9dd24ac306e975391f949b/LayoutTests/accessibility/url-test.html